ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -0,0 +1,151 @@
+import { Tab, TabHeading, Tabs, Text } from "native-base";
+import * as React from "react";
+import { Animated, StyleSheet } from "react-native";
+import { NavigationScreenProps } from "react-navigation";
+import { ScreenContentHeader } from "../../components/screens/ScreenContentHeader";
+import TopScreenComponent from "../../components/screens/TopScreenComponent";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+
+type Props = NavigationScreenProps;
+
+type State = {
+  currentTab: number;
+};
+
+// Scroll range is directly influenced by floating header height
+const SCROLL_RANGE_FOR_ANIMATION = 72;
+
+const styles = StyleSheet.create({
+  tabBarContainer: {
+    elevation: 0,
+    height: 40
+  },
+  tabBarContent: {
+    fontSize: customVariables.fontSizeSmall
+  },
+  tabBarUnderline: {
+    borderBottomColor: customVariables.tabUnderlineColor,
+    borderBottomWidth: customVariables.tabUnderlineHeight
+  },
+  tabBarUnderlineActive: {
+    height: customVariables.tabUnderlineHeight,
+    // borders do not overlap eachother, but stack naturally
+    marginBottom: -customVariables.tabUnderlineHeight,
+    backgroundColor: customVariables.contentPrimaryBackground
+  },
+  searchDisableIcon: {
+    color: customVariables.headerFontColor
+  }
+});
+
+const AnimatedTabs = Animated.createAnimatedComponent(Tabs);
+/**
+ * A screen that contains all the Tabs related to services.
+ */
+class ServicesHomeScreen extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      currentTab: 0
+    };
+  }
+
+  private animatedScrollPositions: ReadonlyArray<Animated.Value> = [
+    new Animated.Value(0),
+    new Animated.Value(0),
+    new Animated.Value(0)
+  ];
+
+  // tslint:disable-next-line: readonly-array
+  private scollPositions: number[] = [0, 0, 0];
```

Comment:
```suggestion
  private scrollPositions: number[] = [0, 0, 0];
```
I also checked **MessagesHomeScreen** and I noticed this class member is never used. Consider to remove it if it won't be useful for next developments

Reply:
@Undermaken Yes it is not necessary at the moment. I can remove it and add it when the list of services is integrated